src/shared/components/barGraph/BarGraph.tsx

```diff
@@ -0,0 +1,203 @@
+import * as React from "react";
+import * as _ from 'lodash';
+import {ThreeBounce} from 'better-react-spinkit';
+import { CancerStudy } from 'shared/api/generated/CBioPortalAPI';
+import * as Highcharts from 'highcharts';
+
+import HCE from 'highcharts/modules/exporting';
+HCE(Highcharts);
+
+
+export interface IBarGraphProps {
+    data:CancerStudy[];
+};
+
+interface ICancerTypeStudy {
+    caseCount:number;
+    shortName:string;
+    studies: CancerStudy[];
+};
+
+interface ICounterObj {
+    [name:string]: ICancerTypeStudy;
+};
+
+export default class BarGraph extends React.Component<IBarGraphProps, {colors: string[]}> {
+
+    constructor() {
+        super();
+    }
+
+    getShortName(shortName:string) {
```

Comment:
could you use a map instead of a switch statement.  If shortName is a key in the map, then return value, otherwise, just return shortName